docs(time picker): update time picker documentation (#1887)
IgniteUI/igniteui-docfx: en/components/time_picker.md

### Usage

> The first step is to import the `IgxTimePickerModule` in the **app.module.ts** file. Note that the IgxTimePicker is also dependent on the **BrowserAnimationsModule** and on the **HammerModule** for touch interactions, so they need to be added to the AppModule as well:

Reviewer: Make this a proper `Note` decorated section.

```suggestion
The first step is to import the `IgxTimePickerModule` in the **app.module.ts** file. 

> [!NOTE]
> The IgxTimePicker is also dependent on both the **BrowserAnimationsModule** and **HammerModule** for touch interactions. They need to be added to the AppModule as well.
```

> <igx-time-picker ></igx-time-picker>

Reviewer: ```suggestion
<igx-time-picker></igx-time-picker>
```

```

```html

> <igx-time-picker [itemsDelta]="{hours:1, minutes:5}" format="HH:mm tt" [vertical]="true" [minValue]="min" [maxValue]="max" (onValidationFailed)="onValidationFailed($event)"></igx-time-picker>

Reviewer: Difficult to read, move each property to a new line.
